### DISCUSSION TITLE
how to exclude a custom element from vite project

### DISCUSSION THREAD

**Discussion by jphunt** [quasarframework/quasar#15520] [General - Components / Directives / etc]
I have a custom element from old project,  "magnum-interview",
I am using inside our project, I got "If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement." error.

Project is created from quasar cli  with vite
Don't know how to exclude custom tag inside "quasar.config.js"

Tried this, doesn't work

viteVuePluginOptions: {
        exclude: ['magnum-interview']
      },

**Comment by dongwa** [comment]
```js
viteVuePluginOptions: {     
 template: {
        compilerOptions: {
          isCustomElement: (tag) => tag.includes('magnum-interview')
        }
      }
},
```
